app/assets/uswds/_uswds-theme.scss

```diff
@@ -64,3 +64,12 @@ Add a list of changed settings in the form $setting: value.
 .usa-footer__secondary-section .usa-social-link:hover .usa-social-link__icon {
   background-color: white;
 }
+
+.dashboard-container {
+  @media (min-width:1025px) {
+    // pushes the last child to the left if it's odd
+    :nth-last-child(1):nth-child(odd) {
+      margin-right: calc(45% - 1rem)
+    }
```

Comment:
shouldn't this use the flex wrapping rules instead of trying to calculate a margin? that's seems funky to me

Reply:
you're right, I removed this hacky crap. the confusing thing was figuring out how to use USWDS' grid system which wants everything explicitly broken into rows - this was tricky to do when we have an unknown amount of cards. I figured out a solution which uses ruby's each_slice method to break the card data into hypothetical row pairs.